### DISCUSSION TITLE
Twin Notes Not Appearing

### DISCUSSION THREAD

**Discussion by jperkinsollila** [LumpBloom7/sentakki#582] [General]
Even in experimental if I enable twin notes and twin slides unless I also use the old converter none will appear unless the map is a gimmick map with simultaneous presses.

**Comment by LumpBloom7** [comment]
Yes. This is intentional, I believe it is mentioned in the tooltip that appears when hovering them.

The reason is due to the new (now default) converter not being designed to generate twin notes due to the difficulties in making it feel good.

**Reply by LumpBloom7** [reply]
This has since been resolved with the addition of twin notes for the default converter. This can be accessed via the Experimental mod.
